.github/actions/build_oci_image/action.yaml

```diff
@@ -23,6 +23,7 @@ runs:
       id: build-image
       uses: docker/build-push-action@v2
       with:
+        context: ./docker/app
```

Comment:
This file is a candidate to be starndard for all the charms. Can't you pass the full path of the dockerfile instead in the "file" param and move the files inside docker/app to root?

Reply:
That wouldn't be possible since the dockerfile needs to copy a folder inside ./docker/app as part of the building process. I could modify the dockerfile in that case, so instead of having COPY app . we put the path there. Any thoughts?